src/aliceVision/sfmData/View.hpp

```diff
@@ -32,6 +33,18 @@ enum class EEXIFOrientation
   , UNKNOWN = -1
 };
 
+struct GPSExifTags
```

Comment:
@fabiencastan I don't know if it is the correct or proper way to do it, I just wanted to avoid repeating the raw strings for the GPS tags in many parts of the code (better maintainability).

Reply:
I would keep it shorter:
````
struct GPSExifTags
{
    static const std::string latitude = "GPS:Latitude";
...
```
And "static const std::string" could be replaced more properly by "constexpr std::string" with C++20.

Reply:
yes I thought about that at first, the only thing is that you cannot initialize the static member inline, you have to do it in the cpp. I went for the functions to avoid any sort of problems with the initialization etc
But I can go back to it if it is better

Reply:
Yes, you're right. We will change it one day when we will move to C++20 ;)